core/stringUtils.js

```diff
@@ -33,6 +32,14 @@ const normalizeName = R.pipe(
 
 const capitalizeFirstLetter = text => text.charAt(0).toUpperCase() + text.slice(1)
 
+const removeNewLines = R.when(
+  isString,
+  R.pipe(
+    R.split(/\r\n|\r|\n/g),
+    R.join(' ')
+  )
+)
+
```

Comment:
should it be just a string replace instead?

Reply:
I don't think so. It's really a remove new lines. I added a space for making the value more readable